.vsts-compliance.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt in the project root for license information.
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
```

Comment:
Not now, but probably should rename master -> main with these assortment of repos at some point.

Reply:
Logged a bug in our internal Azure DevOps system to track this. It should've been done when we did all the other renames